job-dsls/jobs/scrclr_scan_seed_job.groovy

```diff
@@ -0,0 +1,25 @@
+job('srcclr_scan_seed_job') {
+    description('Scan job, which generates scanning jobs for upstream projects')
+    parameters {
+        stringParam('REPO_FILE_URL','https://raw.githubusercontent.com/akoufoudakis/droolsjbpm-build-bootstrap/BXMSPROD-533/script/repository-list.txt','URL of the rpository-list.txt file')
+        stringParam('KIE_JENKINS_SCRIPTS_REPO', 'https://github.com/akoufoudakis/kie-jenkins-scripts.git', '')
```

Comment:
This should probably be fixed before merge.

Reply:
What exactly do you mean? To remove the default values?

Reply:
this is pointing to your repo, we should point to kiegroup